packages/bot-web-ui/src/components/dashboard/dashboard.tsx

```diff
@@ -266,14 +265,14 @@ const Dashboard = ({
                         onTabItemClick={handleTabChange}
                         top
                     >
-                        <div icon='IcDashboardComponentTab' label={localize('Dashboard')} id='id-dbot-dashboard'>
+                        <div icon='IcDashboardComponentTab' label={localize('Dashboard')} id={TAB_IDS[active_tab]}>
                             <DashboardComponent />
                         </div>
-                        <div icon='IcBotBuilderTabIcon' label={localize('Bot Builder')} id='id-bot-builder' />
-                        <div icon='IcChartsTabDbot' label={localize('Charts')} id='id-charts'>
+                        <div icon='IcBotBuilderTabIcon' label={localize('Bot Builder')} id={TAB_IDS[active_tab]} />
+                        <div icon='IcChartsTabDbot' label={localize('Charts')} id={TAB_IDS[active_tab]}>
                             <Chart />
                         </div>
-                        <div icon='IcTutorialsTabs' label={localize('Tutorials')} id='id-tutorials'>
+                        <div icon='IcTutorialsTabs' label={localize('Tutorials')} id={TAB_IDS[active_tab]}>
```

Comment:
@maryia-matskevich-deriv we are passing the IDs dynamically, this will cause duplicating the ids,
`TAB_IDS[active_tab]` will result `tab-charts` for all of the tabs. kindly refer to the screenshot.

<img width="1787" alt="image" src="https://user-images.githubusercontent.com/90243468/229423056-34e97479-0059-4f16-b8ed-88bc4315ecbf.png">

Reply:
thank you that you pay attention